resources/multisite_drupal_standard.make

```diff
@@ -789,15 +789,7 @@ projects[workbench_og][version] = "2.0-beta1"
 projects[workbench_og][patch][] = https://www.drupal.org/files/issues/workbench_og-node_access-2835937.patch
 
 projects[wysiwyg][subdir] = "contrib"
-projects[wysiwyg][version] = "2.2"
-projects[wysiwyg][patch][] = patches/wysiwyg-ckeditor4-bug-version-1799.patch
```

Comment:
Why we remove this patch? it's not included in the version 2.4.

Reply:
It is included here 
http://cgit.drupalcode.org/wysiwyg/tree/editors/ckeditor.inc?h=7.x-2.4#n162

Reply:
Actually it's no longer necessary since the range of version has been upgraded in 2.4, the version we use is supported.
But, there is still a warning message showing in our backend due to the fact we never changed the preferred version of the profile. I have added a hook_update in multisite_wysiwyg feature to fix this.